include/zephyr/drivers/clock_control/clock_control_tcc_ccu.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 Hounjoung Rim <hounjoung@tsnlab.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_CLOCK_CONTROL_CLOCK_CONTROL_TCC_CCU_H_
+#define ZEPHYR_INCLUDE_DRIVERS_CLOCK_CONTROL_CLOCK_CONTROL_TCC_CCU_H_
+
+/*
+ * DEFINITIONS
+ */
+
+#define CKC_MHZ(x) (uint32_t)((x) * 1000000)
```

Comment:
Please remove it, we have `DT_FREQ_M`.
(or you can define an alias for it. It can be called `CKC_MHZ`.)